docs(blog): Maker announcement (#161)
spokestack/spokestack-website: content/blog/introducing-spokestack-maker/index.md

---
title: Introducing Spokestack Maker - AutoML Personal Voice for Creators
date: '2021-03-25'

> description: Spokestack Maker introduces sophisticated AutoML tools for voice interface to a new audience of prototypers, enthusiasts, and makers.

Reviewer: ```suggestion
description: Spokestack Maker introduces sophisticated AutoML tools for voice interfaces to a new audience of prototypers, enthusiasts, and makers.
```

tags: News, Product
author: noel
draft: false
hero:
---

> Spokestack exists to help put voice into software, because voice is a humane way to interface with software, and because software needs better, more accessible tools to utilize voice.

Reviewer: Idk if "humane" is the right word. Did you mean "human"?

```suggestion
Spokestack exists to help put voice into software because voice is a humane way to interface with software and because software needs better, more accessible tools to utilize voice.
```

Author: "humane" is intentional

> [Spokestack Maker](/account/settings#billing) introduces sophisticated AutoML tools for voice to a new audience of prototypers, enthusiasts, and makers. Create, train, distribute, and use state-of-the-art personal wakeword, keyword, and text to speech models — with _your_ own voice.

Reviewer 2: Seeing all these links to the billing page makes me think we should have a Maker splash page with a big CTA button—or is that already in the works?

Author: In the works, but figured this would be helpful to get done as a reference + jumpstart thinking on what we want to say externally.

> [Spokestack Maker](/account/settings#billing) introduces sophisticated AutoML tools for voice to a new audience of prototypers, enthusiasts, and makers. Create, train, distribute, and use state-of-the-art personal wakeword, keyword, and text to speech models — with _your_ own voice.

Reviewer: ```suggestion
[Spokestack Maker](/account/settings#billing) introduces sophisticated AutoML tools for voice to a new audience of prototypers, enthusiasts, and makers. Create, train, distribute, and use state-of-the-art personal wake word, keyword, and text-to-speech (TTS) models — with _your_ own voice.
```

> [Spokestack Maker](/account/settings#billing) introduces sophisticated AutoML tools for voice to a new audience of prototypers, enthusiasts, and makers. Create, train, distribute, and use state-of-the-art personal wakeword, keyword, and text to speech models — with _your_ own voice.

Reviewer: I think this is the first place you mention "personal" models - I'd link your other blog post here

```suggestion
[Spokestack Maker](/account/settings#billing) introduces sophisticated AutoML tools for voice to a new audience of prototypers, enthusiasts, and makers. Create, train, distribute, and use state-of-the-art personal wakeword, keyword, and text to speech models — with _your_ own voice.
```

Spokestack already offers powerful, industry-leading libraries for utilizing voice in [Python](/docs/python/getting-started), [React Native](/docs/react-native/getting-started), [Android](/docs/android/getting-started), and [iOS](/docs/ios/getting-started). With [Spokestack Maker](/account/settings#billing), that voice becomes personal and customizable, utilizing state of the art self-service AutoML technology.

> With [Spokestack Maker](/account/settings#billing), you have the power of three new tools for training your own [wake word](/docs/concepts/wakeword), [keyword](/docs/concepts/keyword), and [TTS](/docs/concepts/tts) models using data you record yourself. In addition, you'll utilize the same NLU model trainer and TTS showcase available to all existing Spokestack Free accounts.

Reviewer 2: ```suggestion
With [Spokestack Maker](/account/settings#billing), you have the power of three new tools for training your own [wake word](/docs/concepts/wakeword), [keyword](/docs/concepts/keyword), and [TTS](/docs/concepts/tts) models using data you record yourself. You'll still have access to the same NLU model trainer and TTS showcase available to all existing Spokestack Free accounts.
```

> With [Spokestack Maker](/account/settings#billing), you have the power of three new tools for training your own [wake word](/docs/concepts/wakeword), [keyword](/docs/concepts/keyword), and [TTS](/docs/concepts/tts) models using data you record yourself. You'll still have access to the same NLU model trainer and TTS showcase available to all existing Spokestack Free accounts.

Reviewer: Unless we're committing to "Spokestack Free" as a branded name of some sort?

```suggestion
With [Spokestack Maker](/account/settings#billing), you have the power of three new tools for training your own [wake word](/docs/concepts/wakeword), [keyword](/docs/concepts/keyword), and [TTS](/docs/concepts/tts) models using data you record yourself. You'll still have access to the same NLU model trainer and TTS showcase available to all existing free accounts.
```

> Spokestack's Maker subscription is designed for two audiences: hobbyists who want to personalize their projects, and developers who want to prototype a project as realistically as possible before committing to training a universal wake word model or studio-quality TTS voice.

Reviewer: Idk if you need this paragraph - I think it's copy and pasted from your other article?
```suggestion

```

Author: lol guilty, but I wanted to address the "who" question explicitly.

> Want to test drive powerful machine learning voice features like keyword recognition, wake word recognition, and custom AI voices without breaking the bank? [Spokestack Maker](/account/settings#billing) brings these features to the a creator market that has never been accessible before.

Reviewer: ```suggestion
Want to test drive powerful machine learning voice features like keyword recognition, wake word recognition, and custom AI voices without breaking the bank? [Spokestack Maker](/account/settings#billing) brings these features to a creator market that has never been accessible before.
```

> You'll be able to download your personal models, train and retrain your personal models and enjoy API access your personal TTS voice as long as your Maker subscription is active. If you decide you'd like to take it to the next level and make a universal wake word/keyword model or a studio-quality TTS voice, just [let us know](mailto:hello@spokestack.io?subject=I want to know more about Spokestack'suUniversal voice models)!

Reviewer 2: ```suggestion
You'll be able to download your personal models, train and retrain your personal models and enjoy API access your personal TTS voice as long as your Maker subscription is active. If you decide you'd like to take it to the next level and make a universal wake word/keyword model or a studio-quality TTS voice, just [let us know](mailto:hello@spokestack.io?subject=I want to know more about Spokestack's Universal voice models)!
```
